containerm/ctr/ctrd_client_test.go

```diff
@@ -596,10 +596,10 @@ func TestAttachContainer(t *testing.T) {
 				}()
 
 				mockIoMgr.EXPECT().GetIO(testCtr.ID).Return(mockIO)
+				mockIO.EXPECT().Stream().Return(mockStream)
 				mockReadCloser.EXPECT().Read(gomock.Any()).DoAndReturn(func(p []byte) (int, error) {
 					return -1, io.EOF
 				})
```

Comment:
```suggestion
				mockReadCloser.EXPECT().Read(gomock.Any()).DoAndReturn(func(p []byte) (int, error) {
					return -1, io.EOF
				})..AnyTimes()
```
This is done in a separate  go routine and sometimes the test fails as it is not called on time. By adding AnyTimes those occasional fails would be eliminated.